releasenotes/notes/Allow-DBM-auto-discovery-to-optionally-set-region-d3006501542da7bf.yaml

```
@@ -0,0 +1,7 @@
+---
+enhancements:
+  - |
+    This change allows DBM auto-discovery users to optionally set the region for where their aurora clusters are running.
+    This is can be used in lieu of relying on IMDS to discover the region through instance metadata. This is a nicer experience for users
```

```suggestion
    This can be used in lieu of relying on IMDS to discover the region through instance metadata. This is a nicer experience for users
```
Looks like an extra `is` made it's way in